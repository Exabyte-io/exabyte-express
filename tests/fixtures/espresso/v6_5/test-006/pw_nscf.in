version https://git-lfs.github.com/spec/v1
oid sha256:5b4aa5225d9614aaa4f388b4d5d20a379287618456d2413ee7112036a25cda9e
size 1037
