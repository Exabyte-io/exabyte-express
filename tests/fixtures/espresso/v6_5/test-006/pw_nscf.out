version https://git-lfs.github.com/spec/v1
oid sha256:edc3dfb19b2e2e390b72131c3c37ec3329ec6d8fe62291803bc2a23b54942014
size 8494
